src/ui/osx/TogglDesktop/Feature/Timeline/Views/TimelineCollectionView.swift

```diff
@@ -23,9 +23,10 @@ final class TimelineCollectionView: NSCollectionView {
 
     override func mouseDown(with event: NSEvent) {
         super.mouseDown(with: event)
-        handleMouseClick(with: event)
+        if #available(OSX 10.13, *) {
```

Comment:
I deliberately disable this feature on 10.11 and 10.12 since it causes a lot of troubles, which make the Editor randomly closes and opens.

I guess it is the same reason how 10.11 handle the Mouse action and the Popover's behavior.

We encountered this issue before in the Time Entry List.

https://github.com/toggl-open-source/toggldesktop/blob/2be885a2276ab736abc1767b3e5f26748d3cf904/src/ui/osx/TogglDesktop/Feature/TimeEntry/Views/TimeEntryCollectionView.m#L56-L57